CMakeModules/build_glbinding.cmake

```diff
@@ -0,0 +1,52 @@
+INCLUDE(ExternalProject)
+
+SET(prefix ${CMAKE_BINARY_DIR}/third_party/glb)
+
+SET(LIB_POSTFIX "")
+IF (${CMAKE_BUILD_TYPE} STREQUAL "Debug")
+    SET(LIB_POSTFIX "d")
+ENDIF()
+
+SET(glbinding_location
+    ${prefix}/lib/${CMAKE_SHARED_LIBRARY_PREFIX}glbinding${LIB_POSTFIX}${CMAKE_SHARED_LIBRARY_SUFFIX})
+
+IF(CMAKE_VERSION VERSION_LESS 3.2)
+    IF(CMAKE_GENERATOR MATCHES "Ninja")
+        MESSAGE(WARNING "Building with Ninja has known issues with CMake older than 3.2")
+    endif()
+    SET(byproducts)
+ELSE()
+    SET(byproducts BYPRODUCTS ${glbinding_location})
+ENDIF()
+
+IF(UNIX)
+    SET(CXXFLAGS "${CMAKE_CXX_FLAGS} -w -fPIC")
+    SET(CFLAGS "${CMAKE_C_FLAGS} -w -fPIC")
+ENDIF(UNIX)
+
+ExternalProject_Add(
+    glb-ext
+    GIT_REPOSITORY https://github.com/cginternals/glbinding.git
+    GIT_TAG v2.0.0
```

Comment:
I have seen v2.0.1 and tried it out, i faced some issues while compiling against that version. This [commit](https://github.com/cginternals/glbinding/commit/10bdcdef15256a5cd9c181ae928c22b35ae77a58) from v2.0.1 is conflicting with standard `gl.h` header. I haven't figured how to resolve that elegantly yet.